Books/render-books.js

```diff
@@ -0,0 +1,58 @@
+import { findById } from './common/utils.js';
+
+export default function readMoreBooks(book) {
+    
+    const li = document.createElement('li');
+    li.className = book.category;
+    li.title = book.description;
+
+    const h3 = document.createElement('h3');
+    h3.textContent = book.name;
+    li.appendChild(h3);
+
+    const img = document.createElement('img');
+    img.src = '../assets/' + book.image;
+    img.alt = book.name + 'image';
+    li.appendChild(img);
+
+    const p = document.createElement('p');
+    p.className = 'price';
+    const ItemPrice = '$' + book.price.toFixed(2);    
+    p.textContent = ItemPrice;
+
+    const addButton = document.createElement('button');
+    addButton.textContent = 'Add';
+    addButton.value = book.id;
+    addButton.addEventListener('click', () => {
+
+        let json = localStorage.getItem('CART');
+        let cart;
+        if (json) {
+            cart = JSON.parse(json);
+        } else {
+            cart = [];
+        }
+
+        let cartItem = findById(book.id, cart);
```

Comment:
Looks like you're trying to find by id here but I don't see that any of your book objects have an id key. If you fix that I think you site should work.